code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -170,6 +172,22 @@
 /// If you wanna try someday feel free, but I can't manage it
 /datum/plane_master_group/popup
 
+/// This is janky as hell but since something changed with CENTER positioning after build 1609 we have to switch to the bandaid LEFT,TOP positioning
+/// using LEFT,TOP at or before 1609 will result in another broken offset for cameras
+#define MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS 1609
+
+/datum/plane_master_group/popup/attach_to(datum/hud/viewing_hud)
+	// If we're about to display this group to a mob who's client is more recent than the last known version with working CENTER, then we need to remake the relays
+	// with the correct screen_loc using the relay override
+	if(viewing_hud.mymob?.client?.byond_build > MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS)
+		for(var/thing in plane_masters)
+			var/atom/movable/screen/plane_master/plane = plane_masters[thing]
```

Comment:
```suggestion
```
Not required when you apply the above review

Reply:
Multiline comments are a thing, click drag the line number to apply a comment on multiple lines